datadog/api/api_client.py

```diff
@@ -44,8 +43,8 @@ def _get_http_client(cls):
         return cls._http_client
 
     @classmethod
-    def submit(cls, method, path, body=None, attach_host_name=False, response_formatter=None,
-               error_formatter=None, **params):
+    def submit(cls, method, path, api_version=_api_version, body=None, attach_host_name=False,
```

Comment:
let's also add `:param api_version ...` in the comments below to be consistent with the other arguments.